src/set_theory/zfc/basic.lean

```diff
@@ -950,6 +954,28 @@ set.ext $ λ z, iff.symm Set.mem_powerset
 set.ext $ λ z, by { refine iff.trans _ Set.mem_sUnion.symm, exact
 ⟨λ ⟨._, ⟨a, rfl, ax⟩, za⟩, ⟨a, ax, za⟩, λ ⟨a, ax, za⟩, ⟨_, ⟨a, rfl, ax⟩, za⟩⟩ }
 
+@[ext] theorem ext {x y : Class.{u}} : (∀ z : Class.{u}, z ∈ x ↔ z ∈ y) → x = y :=
+begin
+  refine λ h, set.ext (λ z, _),
+  change x z ↔ y z,
+  rw [←mem_hom_left z x, ←mem_hom_left z y],
+  exact h z
+end
+
+theorem coe_mem_powerset {x : Class.{u}} {y : Set.{u}} : (powerset x) y ↔ ↑y ⊆ x := iff.rfl
```

Comment:
That looks like defeq abuse. Did you mean this?
```suggestion
theorem coe_mem_powerset {x : Class.{u}} {y : Set.{u}} : (y : Class.{u}) ∈ powerset x ↔ ↑y ⊆ x := iff.rfl
```
Also, is this not generally true?
```suggestion
theorem mem_powerset {x y : Class.{u}} : y ∈ powerset x ↔ y ⊆ x := iff.rfl
```

Reply:
Re def-eq abuse: see the doc note on `Class`. We treat it as `Set → Prop` (as in standard in set-theoretic treatments), and as such it's preferred to write `A x` instead of `x ∈ A`. 

Re `mem_powerset`: this is actually not true. For instance, the class `y` of limit ordinals is a subset of the class of all ordinals, but it does not belong to its powerset class, as members of classes must be sets. By requiring `y` to be a set, we get `coe_mem_powerset`.

Reply:
Okay, then let's get rid of the unnecessary brackets as they cause confusion
```suggestion
theorem coe_mem_powerset {x : Class.{u}} {y : Set.{u}} : powerset x y ↔ ↑y ⊆ x := iff.rfl
```